hack/env.sh

```diff
@@ -9,6 +10,7 @@ if [ -z $SKIP_VAR_SET ]; then
 else
         [ -z $SRIOV_CNI_IMAGE ] && echo "SRIOV_CNI_IMAGE is empty but SKIP_VAR_SET is set" && exit 1
         [ -z $SRIOV_INFINIBAND_CNI_IMAGE ] && echo "SRIOV_INFINIBAND_CNI_IMAGE is empty but SKIP_VAR_SET is set" && exit 1
+        [ -z $RDMA_CNI_IMAGE ] && echo "RDMA_CNI_IMAGE is empty but SKIP_VAR_SET is set" && exit 1
```

Comment:
Same question here. We can change the check like this https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/4b3a309a38d419a86dc7edb0a1bbb78fb13f71ac/hack/env.sh#L15. To allow empty string as a value.

Reply:
done